### PR TITLE
RI events with unknown times now being dealt with

### PR DESCRIPTION
**Pull request by rshorey**

### CODE REVIEW

File: openstates/ri/events.py [hunk 1]
```diff
@@ -59,14 +59,20 @@ def scrape_agenda(self, url, session):
...
             datetime = "%s %s" % ( date, time )
-        if "CANCELLED" in datetime or "Rise of the House" in datetime:
-            # XXX: Do something more advanced.
+        if "CANCELLED" in datetime:
             return
 
+        if "Rise of the" in datetime:
+            datetime = date
+            kwargs["all_day"] = True
+            kwargs["time_text"] = time
```
Comment (paultag) [inline openstates/ri/events.py]:
```diff
@@ -59,14 +59,20 @@ class RIEventScraper(EventScraper, LXMLMixin):
+        if "Rise of the" in datetime:
+            datetime = date
+            kwargs["all_day"] = True
+            kwargs["time_text"] = time
```

I'd just throw this into title myself; what do you think?

Comment (rshorey) [inline openstates/ri/events.py]: you mean the text (eg "Rise of the committee on whatever") in the event title?

Comment (paultag) [inline openstates/ri/events.py]: Yeah I guess that's a dumb idea. Description?

On Mon, Mar 9, 2015 at 2:09 PM, Rachel <notifications@github.com> wrote:

> In openstates/ri/events.py
> <https://github.com/sunlightlabs/openstates/pull/608#discussion_r26059915>
> :
>
> >              return
> >
> > +        if "Rise of the" in datetime:
> > +            datetime = date
> > +            kwargs["all_day"] = True
> > +            kwargs["time_text"] = time
>
> you mean the text (eg "Rise of the committee on whatever") in the event
> title?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sunlightlabs/openstates/pull/608/files#r26059915>.
>



-- 
Paul Tagliamonte
Software Developer | Sunlight Foundation

Comment (rshorey) [inline openstates/ri/events.py]: just pushed a change, it's now in the description, which should read something like "Meeting Notice: Starting at Rise of the Joint Health & Human Services/HEW Hearing" for these weird ones.

**Comment by paultag** [comment]
:+1:, nice, thanks!

**Pull request merged**